conduct.md

```diff
@@ -15,9 +15,10 @@ TODO: Write an introductory paragraph.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not hold onto their opinions to avoid hurting someone's feelings.
+- Students should not talk over others while they are expressing their opinions.
+- Students should not get attached to their ideas and instead be open to more ideas.
+- Students should not do tasks other than the tasks they are assigned.
```

Comment:
I feel like these are more in line with the Communication and Decisions category rather than overall conduct. Maybe the points could focus more on general behavior in the class like
- Students should not disparage others or prevent others from expressing opinions.
- Students should not take criticism personally and should provide constructive criticism to others.

and I think the last point is pretty good but could be rephrased to more clearly express the idea.
- Students should not do group work that is already assigned to someone else.